### ISSUE TITLE
Accessibility (WCAG level A) file input's label on Upload Package page has incorrect 'for' attribute 

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1188]
1.  Log in to nuget.org and click 'Upload Package'
2.  View source
3.  Scrutinize the for attribute on the "Choose a package..." label.

Result:  It's for "PackageFile", but the actual file input doesn't have that name.
Expected:  That's a problem.  Ideally we should have an id defined on the file input and have the label's for attribute be the same.

**Comment by johnataylor** [comment]
We have a new gallery design.
